src/components/PropertyLookup.vue

```diff
@@ -53,7 +53,14 @@ export default Vue.extend( {
 			}
 		},
 		async searchPropertiesOnScroll( options: SearchOptions ): Promise<void> {
-			const searchResults = await this.$store.dispatch( 'searchProperties', options );
+			let searchResults = await this.$store.dispatch( 'searchProperties', options );
+			searchResults = searchResults.map(
```

Comment:
this has the exact same content as the watch. It is better if it's extracted into a function.

Reply:
updated.